perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -938,4 +938,18 @@ public static String cipherInput(String plainText, boolean decrypt) throws Inter
 
 	}
 
+	public static void checkDestinationType(Destination destination) throws WrongPatternException {
+		String destinationType = destination.getType();
+		if (
```

Comment:
There seems to be too much tabulators in my opinion. Please use 1 instead of 2 or even 3.
And also there is probably no need to use new line on the row 943. You can inspire yourself in VoEntryManager on the line:97 method getAllVos.
Thank you.